cub/test/catch2_test_helper.h

```diff
@@ -36,12 +36,12 @@
 
 #include "cuda/std/__cccl/diagnostic.h"
 #include "test_util_vec.h"
-#include <metal.hpp>
 
 #if __CUDACC_VER_MAJOR__ == 11
 _CCCL_NV_DIAG_SUPPRESS(177) // catch2 may contain unused variableds
 #endif // nvcc-11
 
+#include <cuda/std/__type_traits/type_list.h>
```

Comment:
We settled on using the "proper" header in cub / thrust for now and especially in testing, so
```suggestion
#include <cuda/std/type_traits>
```

Also this should be included in `<cuda/std/type_traits>`